### ISSUE TITLE
Issues with poses.txt generated by Kiss-ICP

### ISSUE THREAD

**Issue by SvenMuc** [jbehley/point_labeler#68]
Hi,
I'm trying to use the [kiss-icp](https://github.com/PRBonn/kiss-icp) to generate a poses.txt file for the [point_labeler](https://github.com/jbehley/point_labeler) tool.
The kiss-icp poses_kitt.txt format seems to be correct, and can be read directly into the labeler tool. But the results are looking different. 

**Kiss-ICP Results**
The results are looking pretty good.
![Screenshot from 2023-05-25 08-44-44](https://github.com/jbehley/point_labeler/assets/13385716/1d94762e-3aab-461a-b76c-2b8fef8a8fe2)

**Point Labeler Results using the Kiss-ICP poses**
There seems to be mainly an issue with the heading.
![Screenshot from 2023-05-25 08-43-17](https://github.com/jbehley/point_labeler/assets/13385716/fe7291fe-b2e2-4f41-b2ad-3fb0ee079ac6)

Are there any differences in the coordinate frames or the interpretation of the poses format?
If I use the default camera frame in the calib.txt, everything should be fine, isn't it?

**Comment by jbehley** [comment]
TLDR: Referring to your issue (https://github.com/PRBonn/kiss-icp/issues/167). When KISS does generate the poses in the LiDAR frame, then you have to modify the `calib.txt`such that the Tr is a identity matrix. 

More details:

I pre/post multiply Tr in the loader such that it fits to the KITTI convention that all poses are given in the camera coordinates, therefore, one has to make P_lidar = inverse(Tr) * P_cam * Tr to get the pose in the LiDAR coordinates.

Thus, the easiest way is to modify the `calib.txt` such that `Tr: 1 0 0 0 0 1 0 0 0 0 1 0`.

**Comment by SvenMuc** [comment]
Thanks for your reply.
Yes, that's what I exactly did and was also my expectation.

Maybe the root cause are the exported kiss-icp poses (see https://github.com/PRBonn/kiss-icp/issues/170).
There seem to be a mismatch of the expected coordinate frames.

**Comment by jbehley** [comment]
could you solve the problem?

**Comment by SvenMuc** [comment]
We identified the root cause for the jumps. I occurs everything we faced some frame drops. In that case the matching fails due to the bigger distance.

**Comment by jbehley** [comment]
Thanks for the info and letting us know. Thus, it seems like the issue is solved and can be closed?

**Comment by SvenMuc** [comment]
yes, from side it is solved. Thank you for your support!.
